src/model/import.ts

```diff
@@ -59,6 +59,9 @@ export class ScannedImport implements Resolvable {
   }
 
   resolve(document: Document): Import|undefined {
+    if (!document.analyzer.canResolveUrl(this.url)) {
```

Comment:
Will this cause problems downstream? The unknown import won't be present at all as a Feature

What if instead Import.document was optional?